src/main/flight/pid_init.c

```diff
@@ -263,6 +270,7 @@ void pidInitFeedforwardLpf(uint16_t filterCutoff, uint8_t debugAxis)
         pidRuntime.feedforwardLpfInitialized = true;
         for (int axis = FD_ROLL; axis <= FD_YAW; axis++) {
             pt3FilterInit(&pidRuntime.feedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
+            pt3FilterInit(&pidRuntime.angleFeedforwardPt3[axis], pt3FilterGain(filterCutoff, pidRuntime.dT));
```

Comment:
Adding a lot of smoothing to spread out the feedforward push may work well for certain setups but I don't think it is the right approach for robust control of many different setups. Ideally we should apply a minimal amount of smoothing to derivative elements to avoid noise but no more. I'm not clear on what problem extremely large FF smoothing (much more than required to eliminate noise) is solving?

Reply:
It stops the wobble due to finger tremor that otherwise makes fast angle feedforward unusable.